src/content/docs/usage/server.mdx

```diff
@@ -2,15 +2,289 @@
+
+The Daytona Server configuration can be viewed by using the following command across all operating systems:
+
+```shell
+  daytona server config
+```
+
+The Daytona Server configuration contains the following properties:
+
+- **Server ID**
+  
+    A unique identifier for the Daytona server instance.
+
+    Example: `12a34bc5-d67e-890f-1gh2-34i5jk6l7m89`
+
+- **API URL**
+  
+    The URL endpoint for the Daytona API.
+
+    Example: `https://api-12a34bc5-d67e-890f-1gh2-34i5jk6l7m89.try-eu.daytona.app`
+
+    The API URL property cannot be manually edited, but you can use it to connect to Daytona with other clients.
+
+- **API Port**
+  
+    The port number on which the Daytona API is accessible.
+
+    Example: `3986`
+
+    If you have something running on port 3986, you can change it to a port that you have available.
+
+- **Default Project Image**
+  
+    The default Docker image used for projects.
+
+    Example: `daytonaio/workspace-project:latest`
+
+    The Default Project Image property can be manually edited to use a different default project image.
+
+- **Default Project User**
+  
+    The default user for projects.
+
+    Example: `daytona`
+
+    The Default Project User property can be manually edited to use a different default project user.
+
+- **FRPS Domain**
+  
+    The domain used by the FRP (Fast Reverse Proxy) server.
+
+    Example: `try-eu.daytona.app`
+
+    The FRPS Domain property can be manually edited to use a different FRP domain.
+
+- **FRPS Port**
+  
+    The port used by the FRP (Fast Reverse Proxy) server.
+
+    Example: `7000`
+
+    The FRPS Port property can be manually edited to use a different FRP port.
+
+- **FRPS Protocol**
+  
+    The protocol used by the FRP (Fast Reverse Proxy) server.
+
+    Example: `https`
+
+    The FRPS Protocol property can be manually edited to use a different FRP protocol.
+
+- **Headscale Port**
+  
+    The port number for the Headscale service.
+
+    Example: `3987`
+
+    The Headscale Port property can be manually edited to use a different Headscale port.
+
+- **Binaries Path**
+  
+    The directory path where server binaries are stored.
+
+    Example: `/Users/Library/Application Support/daytona/server/binaries`
+
+    The Binaries Path directory will be created if it does not exist, and the property can be manually edited to use a different binaries path.
+
+- **Log File Path**
+  
+    The directory path where server logs are stored.
+
+    Example: `/Users/Library/Application Support/daytona/server/daytona.log`
+
+    The Log File Path file will be created if it does not exist, and the property can be manually edited to use a different log file path.
+
+- **Builder Image**
+  
+    The Docker image used by the local builder.
+
+    Example: `daytonaio/workspace-project:latest`
+
+    The Builder Image property can be manually edited to use a different builder image.
+
+- **Local Builder Registry Port**
```

Comment:
THis and the `Local Builder Registry Image` props are used/displayed only if `Builder Registry` is set to local. If another container registry is selected, then we display: `Builder Registry: {REGISTRY}`.

Not sure how we could display this here. We might not even have to but just wanted you to know.

Reply:
Thanks for the explanation - I added a note related to local builder registry image and port availability (https://github.com/daytonaio/docs/pull/105/commits/adc4b81ad23d8ed6db36d20e82ffa88cdebe8e2c).